### PR TITLE
Fix convergence rates with 64 bit integer key column

### PR DESCRIPTION
**Pull request by kronbichler**

### CODE REVIEW

**Comment by davydden** [comment]
./run-tests

File: tests/base/convergence_table_05.cc [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2010 - 2015 by the deal.II authors
```
Comment (jppelteret) [inline tests/base/convergence_table_05.cc]: I can never remember - do these dates get auto-corrected by a script before we do a release?

Comment (kronbichler) [reply tests/base/convergence_table_05.cc]: I think so, yes.

Comment (bangerth) [reply tests/base/convergence_table_05.cc]: Yes.

**Pull request merged**